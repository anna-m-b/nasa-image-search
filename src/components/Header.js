import React from 'react';
import nasaLogo from '../images/nasa.jpg';
import '../styles/Header.css';

const Header = () => {
  return (
    <div className="header">
      <img
        width="500px"
        className="header_nasa-logo"
        src={nasaLogo}
        alt="Nasa Logo"
      />
    </div>
  );
};

export default Header;
