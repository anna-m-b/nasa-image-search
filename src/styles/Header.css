.header {
  /* border: 1px solid purple; */
}
.header_nasa-logo {
  width: 500px;
}