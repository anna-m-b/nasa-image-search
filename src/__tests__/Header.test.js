import React from 'react';
import { render } from '@testing-library/react';
import Header from '../components/Header';

describe('Header', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('it renders the nasa logo', () => {
    const { getByAltText } = render(<Header />);
    expect(getByAltText(/nasa logo/i)).toBeInTheDocument();
  });
});
