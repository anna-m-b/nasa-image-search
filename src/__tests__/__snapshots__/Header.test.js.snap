// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <img
      alt="Nasa Logo"
      class="header_nasa-logo"
      src="nasa.jpg"
      width="500px"
    />
  </div>
</DocumentFragment>
`;
